src/field_message_box.c

```diff
@@ -0,0 +1,162 @@
+#include "global.h"
+#include "string.h"
+#include "string_util.h"
+#include "task.h"
+#include "text.h"
+
+extern void sub_81973A4(void);
+extern void sub_81973C4(u8, u8);
+extern u16 sub_8197224(void);
+extern int sub_8196094(void);
+extern void sub_8196080(u8*);
+extern void AddTextPrinterForMessage(u8);
+extern void sub_8197434(u8, u8);
+extern void sub_81973FC(u8, u8);
```

Comment:
All these are either already in new_menu_helpers.h or should be (yes, even the Battle Frontier ones, it was a bad split)

Reply:
just the last four or all of the functions

Reply:
All of them, I couldn't find any that weren't in new_menu_helpers or battle_frontier_1